x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/AuthorizationServiceTests.java

```diff
@@ -274,7 +276,8 @@ public void setup() {
             null, Collections.emptySet(), licenseState, new IndexNameExpressionResolver(new ThreadContext(Settings.EMPTY)));
     }
 
-    private void authorize(Authentication authentication, String action, TransportRequest request) {
+    private void authorize(Authentication authentication, String action, TransportRequest request,
```

Comment:
Similarly to the changes in the `AuthenticationServiceTests`, the tests in this class have been modified to assert that the `requestId` is set following a successful authz.